tla-types/src/test/scala/at/forsyte/apalache/tla/typecomp/BuilderTest.scala

```diff
@@ -51,9 +51,24 @@ trait BuilderTest extends AnyFunSuite with BeforeAndAfter with Checkers with App
 
     def minIntGen(min: Int) = Gen.choose(min, min + 10)
 
+    val intGen: Gen[Int] = minIntGen(-5)
     val positiveIntGen: Gen[Int] = minIntGen(1)
     val nonnegativeIntGen: Gen[Int] = minIntGen(0)
 
+    val boolGen: Gen[Boolean] = Gen.oneOf(true, false)
+
+    // empty strings mess up regex matching for uninterpreted literals
+    val strGen: Gen[String] = Gen.alphaStr.suchThat(_.nonEmpty)
```

Comment:
I think we can name this more aptly

```suggestion
    val nonEmptyStrGen: Gen[String] = Gen.alphaStr.suchThat(_.nonEmpty)
```